OpenRA.Mods.Common/Traits/CustomSelectionSize.cs

```diff
@@ -31,5 +32,13 @@ public int2 SelectionSize(Actor self)
 		{
 			return new int2(info.CustomBounds[0], info.CustomBounds[1]);
 		}
+
+		public int2 SelectionOffset(Actor self)
+		{
+			if (info.CustomBounds.Length > 2)
+				return new int2(info.CustomBounds[2], info.CustomBounds[3]);
+			else
+				return int2.Zero;
```

Comment:
This could be simplified to `return info.CustomBounds.Length > 2 ?  new int2(info.CustomBounds[2], info.CustomBounds[3]) : int2.Zero`.

Reply:
Changed to a one-line as suggested.